Examples/Lenses/AutoLenses.stencil

```diff
@@ -0,0 +1,38 @@
+infix operator *~: MultiplicationPrecedence
```

Comment:
This is really cool. I also implemented "Bound Lenses", à la: https://github.com/narfdotpl/lenso, but using Sourcery :). It's lenses with an OO-like syntax.